llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir

```diff
@@ -0,0 +1,412 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# Test wave32
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx940 -mattr=+wavefrontsize32 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW32,GFX940 %s
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1100 -mattr=+wavefrontsize32 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW32,GFX11 %s
```

Comment:
GFX11 defaults to the MUBUF ABI, this is not using FLATSCR

Reply:
should I use MUBUFW32?

Reply:
yes